src/components/Footer/Footer.js

```diff
@@ -2,7 +2,70 @@ import React from 'react';
 import './Footer.scss';
 
 const Footer = () => {
-  return <div>Footer 입니다.</div>;
+  return (
+    <div className="footerWrapper">
+      <div className="footerDiv">
+        <div className="logoImg" />
+        <div className="webInfo">
+          <ul className="cont1">
+            {INFO_LIST.map(a => {
```

Comment:
```suggestion
            {INFO_LIST.map(a => {
```
매개변수 a 에는 어떤 데이터가 들어오나요?

Reply:
해당 홈페이지의 처리약관 및 서비스 약관에 관한 내용을 볼수 있는 링크로 이동해주는 태그들이 존재합니다. 알아보기 쉬운 이름으로 수정하였습니다.